src/cli/cli-definitions.js

```diff
@@ -174,5 +174,15 @@ export default {
   "verbose": {
     env: "VERBOSE",
     help: "Set the logging to verbose"
+  },
+  "logLevel": {
+    help: "Sets the log level"
+  },
+  "enableConfigChanges": {
+    env: "ENABLE_CONFIG_CHANGES",
```

Comment:
Only concern with removing the setting is that enabling config changes adds an extra db call per request.  The setting is the way to provide a simple switch to opt out of that penalty.

I'm happy to default to true, whether that's by removing the setting or otherwise.